client/controllers/map.js

```diff
@@ -20,14 +25,35 @@ Template.map.onCreated(function () {
 });
  
 Template.map.onRendered(function () {
+  
   L.Icon.Default.imagePath = '/packages/bevanhunt_leaflet/images/';
   var map = L.map('map');
-  L.tileLayer.provider('Stamen.Watercolor').addTo(map);
+  L.tileLayer.provider('Stamen.Terrain').addTo(map);
   map.setView([40.077946, -95.989253], 4);
+
+  this.autorun(() => {
+
+    if(!_.isUndefined(this.data) && !_.isUndefined(this.data.airportId) )  {
+      // When we search on a state or anything other than a single airport code, (more than one airport), will have to update this.
+      const airports = Locations.find({ type: 'Airport', airportIds: this.data.airportId() }).fetch();
```

Comment:
I would call this variable locations because eventually it will be states and other types of region as well.